lib/unit/models/api_token/customer/create_customer_token_request.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Request to create a customer token
+# @see https://docs.unit.co/customer-api-tokens#customers-create-customer-bearer-token
+module Unit
+  module ApiToken
+    module Customer
+      class CreateCustomerTokenRequest
+        attr_reader :customer_id, :scope, :verification_token, :verification_code, :expires_in, :resources
+
+        # @param customer_id [String]
+        # @param scope [String]
```

Comment:
this should be an array of strings and we create a string with spaces as separated, like includes but with spaces.

Reply:
"scope": "customers accounts" not array